tool0&1.py

```diff
@@ -0,0 +1,83 @@
+import geopandas as gpd
```

Comment:
Let's have a different file for each program.  Let's use short, descriptive names for the programs, without special characters.